ProMotion.gemspec

```diff
@@ -25,6 +25,9 @@ Gem::Specification.new do |gem|
   gem.version       = ProMotion::VERSION
 
   gem.add_runtime_dependency("methadone", "~> 1.7")
+  gem.add_runtime_dependency("motion_print")
+
+  gem.add_development_dependency("motion_print", "~> 0.0.5")
```

Comment:
probably will set this to be whatever version is released with the force color (0.0.6?)